src/game/game.c

```diff
@@ -1,23 +1,187 @@
 #include "../../include/game/game.h"
 
-/*
- * Todo: implement player_move and complete the description for this function, after completed, please delete todo.
- * @Author: Tingyu Ye,
```

Comment:
Please keep the comment, and fill the description

Reply:
I have commented the game.c